README.md

```diff
@@ -89,9 +85,57 @@ affect_receive:
 
 This config injects a fault to node B at time 20 to delay its receipt of messages on `topic1`. Please see this [fault_injection.py](graph_generator/fault_injection.py) for a full list of faults supported.
 
-## Example Configs And Dataset
+## Example Configs
+
+In the [config](graph_generator/config/autonomous_vehicle/) folder, we have created an example graph that approximates autonomous vehicle software architecture. You can visualize how faults propagate with the following command:
+
+```bash
+bazel run //graph_generator:main -- --graph graph_generator/config/autonomous_vehicle/graph.yaml --node_feature_output ~/out --fault graph_generator/config/autonomous_vehicle/faults/crash_camera_driver1.yaml --fault_label_output /tmp/a --stop 200 --inject_at 10 --viz
```

Comment:
nit: is the extra `--` before `--graph` intentional?

Reply:
Yep. It's a way to tell bazel to pass argument to python instead of bazel itself